resources/js/mixins.js

```diff
@@ -13,5 +13,15 @@ Vue.mixin({
                 return await magento[method]('guest-carts/' + localStorage.mask + '/' + endpoint, data)
             }
         },
+
+        includeTaxAt(location) {
+            return location === true || location === false
+                ? location
+                : (window.config.tax.display[location] ?? 0) >= 1
+        },
+
+        decideTax(including, excluding, location) {
+            return this.includeTaxAt(location) ? including : excluding
+        },
```

Comment:
If a Mixin is used in multiple components my preference is to load that mixin inside those components, like in https://github.com/rapidez/core/blob/master/resources/js/components/Cart/Cart.vue#L2

Reply:
This kinda comes down to the above thing, do we want to overkill with a component above? If that's the solution we choose then I can move this to a dedicated mixin instead.